docs/learn/terra-station/staking.mdx

```diff
@@ -30,7 +30,7 @@ Congratulations, you've just delegated Luna!
 
 ## Withdraw staking rewards
 
-Rewards start accruing the moment you stake Luna. Monitor your rewards in the staking section of Terra Station. Once you have sufficient rewards, follow these steps to withdraw them:
+Rewards start accruing the moment you stake Luna. Monitor your rewards in the staking section of Terra Station. Once you have sufficient rewards, follow the below steps to withdraw them.
```

Comment:
Should be "the steps below"

Not a hard and fast grammar rule, but more readable IMO.